packages/services/src/ctx/full-viewing-key.ts

```diff
@@ -1,4 +1,6 @@
-import { createContextKey } from '@connectrpc/connect';
+import { Code, ConnectError, createContextKey } from '@connectrpc/connect';
 import { FullViewingKey } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/keys/v1/keys_pb';
 
-export const fvkCtx = createContextKey<FullViewingKey | undefined>(undefined);
+export const fvkCtx = createContextKey<() => Promise<FullViewingKey>>(() =>
+  Promise.reject(new ConnectError('No full viewing key available', Code.FailedPrecondition)),
+);
```

Comment:
What is this doing? This may need a code comment for context.

Reply:
much like the other context keys in this package, this calls `createContextKey` with a type and default value.

`createContextKey` is explained in ConnectRPC documentation.

https://connectrpc.com/docs/node/interceptors#context-keys